src/components/nodes/logical/Conjunction.vue

```diff
@@ -0,0 +1,28 @@
+<template>
+  <div/>
+</template>
+
+<script>
+export default {
+  spec: {
+    id: 'std::conjunction',
+    title: 'Conjunction',
```

Comment:
While I appreciate using the formal names for file names and classes, it may be more appropriate for paleographers to use simpler terms such as "And" and "Or" as node titles.